python-threatexchange/threatexchange/fetcher/fetch_api.py

```diff
@@ -199,3 +204,56 @@ def report_false_positive(
                 tags=set(),
             ),
         )
+
+
+class SignalExchangeAPIWithIterFetch(
+    t.Generic[
+        state.TFetchCheckpoint
+    ],  # TODO move to SignalExchangeAPI in a future diff
+    SignalExchangeAPI,
+):
+    """
+    An alternative fetch_once implementation simplifies tracking state
```

Comment:
docstring meant for `def fetch_once` below?

Reply:
No, the alternative fetch_once implementation is referring to fetch_iter